src/tests/unit/pages/my-certificates/my-certificates.spec.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import { useQuery } from '@apollo/client';
+import MyCertificates from '../../../../pages/my-certificates/my-certificates';
+
+jest.mock('../../../../pages/my-certificates/my-certificates.styles', () => ({
+  useStyles: () => ({})
+}));
+
+jest.mock(
+  '../../../../containers/my-certificates/filled-certificates/filled-certificates.styles',
+  () => ({
+    useStyles: () => ({})
+  })
+);
+
+jest.mock(
+  '../../../../containers/my-certificates/empty-certificates/empty-certificates.styles',
+  () => ({
+    useStyles: () => ({})
+  })
+);
+
+jest.mock(
+  '../../../../containers/my-certificates/certificate-table/certificate-table.styles.js',
+  () => ({
+    useStyles: () => ({})
+  })
+);
+
+jest.mock('@apollo/client');
```

Comment:
If we have some requests in components u wanna test, it is a bad idea to mock Appolo client.

https://www.apollographql.com/docs/react/development-testing/testing/

Reply:
- [ x] FIXED

Thanks a lot for the link. I've seen it before but didn't pay enough attention on it. 
After refactoring "my-certificates.spec" according to the article I could increase the coverage from 70% t0 100%. 

--------BEFORE------------
<img width="809" alt="Screenshot 2022-01-31 at 21 06 21" src="https://user-images.githubusercontent.com/40118760/151856459-057f3da5-86df-4728-b21c-3eb4cedb4442.png">

----------AFTER-----------
![Screenshot 2022-01-31 at 21 01 19](https://user-images.githubusercontent.com/40118760/151856285-4d2e9f1a-8088-4eca-8e18-8a4fee502f2f.png)